cli/js/tests/net_test.ts

```diff
@@ -523,7 +523,7 @@ unitTest(
       try {
         while (true) {
           const nread = await conn.read(p);
-          if (nread === Deno.EOF) {
+          if (nread == null) {
```

Comment:
I checked that `null == 0` is false, so what you've done is correct. But it seems prudent to retain the triple equals here.

Reply:
I personally just default to `==` unless the API has the possibility of two distinct values which require a strict equality check, which no good API should. And I like operators which treat nullish values as the same.

That said, I guess `===` is slightly safer for debugging so I'll make the change.